Core/src/org/sleuthkit/autopsy/datamodel/ContentChildren.java

```diff
@@ -33,17 +33,15 @@
 /**
  * Makes the children nodes / keys for a given content object. Has knowledge
  * about the structure of the directory tree and what levels should be ignored.
- * TODO consider a ContentChildren child factory
  */
 class ContentChildren extends AbstractContentChildren<Content> {
 
     private static final Logger logger = Logger.getLogger(ContentChildren.class.getName());
-    //private static final int MAX_CHILD_COUNT = 1000000;
 
     private final Content parent;
 
     ContentChildren(Content parent) {
-        super(); //initialize lazy behavior
+        super("content_" + Long.toString(parent.getId()));
```

Comment:
I wonder if we should use canonical class names or something else a little more "formal" for these key strings. It seems that whoever adds new Children classes in the future might need to survey all of the key names to see which have been taken...

Reply:
Yeah, I had considered doing something different but went with this because DataResultViewerTable uses the node name (Node.getName()) to communicate with the ChildFactory and AbstractContentNode has always set the name in it's constructor to "content_" + id.